Contents/Code/siteToughLoveX.py

```diff
@@ -0,0 +1,116 @@
+import PAsearchSites
+import PAactors
+
+def search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate,searchSiteID):
+    if searchSiteID != 9999:
+        siteNum = searchSiteID
+    searchString = searchTitle.replace(" ","-").replace(",","").replace("'","").replace("?","").lower().strip()
+    Log("searchString " + searchString)
+    modelsPageSortByLetter = PAsearchSites.getSearchSearchURL(siteNum) + searchTitle[:1] #here we get first letter of searchTitle
+    actorResults = HTML.ElementFromURL(modelsPageSortByLetter)
+    actorPage = actorResults.xpath('//a[contains(@href,"' + searchString + '")]')[0].get('href') # looking for our model
+    searchResults = HTML.ElementFromURL(actorPage) # scene search is carried out through the model page
+
+    for searchResult in searchResults.xpath('//div[@class="content-box"]'):
+        titleNoFormatting = searchResult.xpath('.//h2//span')[0].text_content().strip()
+        sceneUrl = searchResult.xpath('.//a')[0].get('href')
+        curID = sceneUrl.replace('/','_').replace('?','!')
+        scenePage = HTML.ElementFromURL(sceneUrl) # geting releaseDate from scene page
+        releaseDate = parse(scenePage.xpath('//span[@class="date"]')[0].text_content().strip()).strftime('%Y-%m-%d')
+        score = 100 - Util.LevenshteinDistance(searchDate, releaseDate)
+        results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = titleNoFormatting + " [" + PAsearchSites.getSearchSiteName(siteNum) + "] " + releaseDate, score = score, lang = lang))
+    return results
+
+def update(metadata,siteID,movieGenres,movieActors):
+    pageURL = str(metadata.id).split("|")[0].replace('_', '/').replace('!','?')
```

Comment:
Could you add in a `Log('******UPDATE CALLED*******')` right after `def update`

Reply:
added to VogoV too thx